### ISSUE TITLE
Library view toolbar, OPDS button's label is generic "menu"

### ISSUE THREAD

**Issue by danielweck** [edrlab/readium-desktop#102]
(should be something like "OPDS")
Note that the following button says "Add EPUB", and the following *actual* menu button says "menu" :)

Note that although there is currently no visual manifestation of the textual label associated with the button icons (e.g. temporary popup tooltip, or permanent underline), screen readers do pick-up the title of the SVG markup that renders the button.

Related issues: https://github.com/edrlab/readium-desktop/issues/103 and https://github.com/edrlab/readium-desktop/issues/104
